cmd/akamai-purger/main.go

```diff
@@ -43,50 +50,48 @@ type Config struct {
 	Beeline cmd.BeelineConfig
 }
 
+// akamaiPurger is a wrapper for an inner slice of URLs to be purged, protected
+// by a mutex.
```

Comment:
This description isn't really accurate. It's much more than a wrapper for a slice. It's a gRPC server that processes purge requests, and happens to handle them by queueing them for efficiency.

Reply:
Ah, thanks for catching this. I'll go back and add the missing details.